web/client/epics/search.js

```diff
@@ -149,24 +148,9 @@ export const searchItemSelected = action$ =>
                 }
 
                 let bbox = item.bbox || item.properties.bbox || toBbox(item);
-                let mapSize = action.mapConfig.size;
-                // zoom by the max. extent defined in the map's config
-                let newZoom = mapUtils.getZoomForExtent(CoordinatesUtils.reprojectBbox(bbox, "EPSG:4326", action.mapConfig.projection), mapSize, 0, 21, null);
```

Comment:
Please use the maxZoom (21) in this line in the zoomToExtent action (see API).
It's useful in case of points, so test it with single points.